File: sentimental-cash/cash.py
from cs50 import get_float

# Prompt input for amout of changes
while True:
    c = get_float("Change owe: ")
    if c >= 0.01:
        break
# convert amount of change for interger calculation
a = int(c * 100)
# calculating amount of 25 cent
c25 = a // 25
a %= 25

# calculating amount of 10
c10 = a // 10
a %= 10

# calculating amount of 5
c5 = a // 5
a %= 5

# calculating amount of 1
c1 = a

# calculating total number of coins
t = int(c25 + c10 + c5 + c1)

print(f"{t}")

File: cash/cash.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int amount; // ensure only acceptable input
    do
    {
        amount = get_int("Change owe: "); // getting the amount of changes
    }
    while (amount < 1);

    int coin25 = amount / 25; // calculating amount of 25 cent
    amount %= 25;             // finding balance for next calculation

    int coin10 = amount / 10; // calculating amount of 10 cent
    amount %= 10;

    int coin5 = amount / 5; // calculating amount of 5 cent
    amount %= 5;

    int coin1 = amount; // calculating amount of 1 cent

    printf("%i\n", coin25 + coin10 + coin5 + coin1); // calculating total number of coins

    return 0;
}
